src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
> @jsor alright will do tomorrow first thing in the morning!

@nrk Are you still planning to file an issue in the pecl-event repo? Otherwise i can do it.

> @clue the additional warning is emitted even when the exception is caught, see the example in my previous comment.

Yes, but only if the exception bubbles up to the `$this->eventBase->loop()` call. This usually results in an uncaught exception anyway since in almost all cases `$loop->run()` is called at the end of the script.